### PR TITLE
[build_debian]: Install grub-common in the base image

### PR DESCRIPTION
**Pull request by stcheng**
**- How to verify it**
`apt-get install -y grub-common`
output:
```
root@str-s6000-on-2:/home/admin# grub-                                                                   
grub-editenv          grub-macbless         grub-mkimage          grub-mkrescue         grub-script-check
grub-file             grub-menulst2cfg      grub-mklayout         grub-mkstandalone     grub-syslinux2cfg
grub-fstest           grub-mkconfig         grub-mknetdir         grub-mount                             
grub-glue-efi         grub-mkdevicemap      grub-mkpasswd-pbkdf2  grub-probe                             
grub-kbdcomp          grub-mkfont           grub-mkrelpath        grub-render-label                      
```

### CODE REVIEW

**Comment by stcheng** [comment]
This is trying to resolve the following issue:
```
TASK [Set next boot device to ONIE] ********************************************
task path: /var/sonicbld/workspace/Deploy/boot_onie/ansible/boot_onie.yml:18
Thursday 05 October 2017  08:45:27 +0000 (0:00:00.103)       0:00:00.397 ****** 
<10.3.147.195> ESTABLISH SSH CONNECTION FOR USER: admin
<10.3.147.195> SSH: EXEC sshpass -d16 ssh -C -vvv -o ControlMaster=auto -o ControlPersist=120s -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o User=admin -o ConnectTimeout=10 -o ControlPath=/var/sonicbld/.ansible/cp/ansible-ssh-%h-%p-%r 10.3.147.195 '/bin/sh -c '"'"'sudo -H -S  -p "[sudo via ansible, key=cykpmkepgiypqlppzkjboontmbgqpviq] password: " -u root /bin/sh -c '"'"'"'"'"'"'"'"'echo BECOME-SUCCESS-cykpmkepgiypqlppzkjboontmbgqpviq; LANG=C LC_ALL=C LC_MESSAGES=C /usr/bin/python'"'"'"'"'"'"'"'"''"'"''
fatal: [str-s6000-acs-7]: FAILED! => {"changed": true, "cmd": "grub-editenv /host/grub/grubenv set next_entry=ONIE", "delta": "0:00:00.020188", "end": "2017-10-05 08:45:28.115046", "failed": true, "invocation": {"module_args": {"_raw_params": "grub-editenv /host/grub/grubenv set next_entry=ONIE", "_uses_shell": true, "chdir": null, "creates": null, "executable": null, "removes": null, "warn": true}, "module_name": "command"}, "rc": 127, "start": "2017-10-05 08:45:28.094858", "stderr": "/bin/sh: 1: grub-editenv: not found", "stdout": "", "stdout_lines": [], "warnings": []}
```

File: build_debian.sh [hunk 1]
```diff
@@ -207,7 +207,8 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     kexec-tools             \
     less                    \
     unzip                   \
-    gdisk
+    gdisk                   \
+    grub-common
```
Comment (qiluo-msft) [inline build_debian.sh]: suggest install grub2-common

Comment (stcheng) [reply build_debian.sh]: updated.=

**Comment by stcheng** [comment]
wait for checks

**Pull request merged**